### PR TITLE
fix: fix tx list empty problem

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/transactions.ts [hunk 1]
```diff
@@ -34,9 +34,12 @@ export default class TransactionsController {
...
     const { pageNo, pageSize, addresses = '' } = params
 
-    let searchAddresses = addresses.split(',')
+    let searchAddresses = addresses
+      .split(',')
+      .map(addr => addr.trim())
+      .filter(addr => addr !== '')
 
-    if (!searchAddresses.length) {
+    if (searchAddresses.length <= 0) {
```
Comment (ashchan) [inline packages/neuron-wallet/src/controllers/transactions.ts]: When will `length` be less than 0?

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/transactions.ts]: Equals to `=== 0`

Comment (ashchan) [reply packages/neuron-wallet/src/controllers/transactions.ts]: <= 0 and === 0 are different.

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/transactions.ts]: rollback to `!searchAddresses.length`

**Pull request merged**